hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/impl/BlockManagerImpl.java

```diff
@@ -98,6 +98,81 @@ public long putBlock(Container container, BlockData data,
         data, endOfBlock);
   }
 
+  @Override
+  public long putBlockForClosedContainer(Container container, BlockData data, boolean overwriteBcsId)
```

Comment:
This method needs a pretty substantial javadoc covering its intended uses and what is supposed to be set in the block and container data by the caller. For example, block overwrite is automatically handled, and caller is supposed to set used bytes values.